zproject/default_settings.py

```diff
@@ -703,6 +703,9 @@
 TOPIC_SUMMARIZATION_MODEL: str | None = None
 # Price per token for input and output tokens.
 # Unit: USD per 1 billion tokens.
-OUTPUT_COST_PER_GIGATOKEN = 0
-INPUT_COST_PER_GIGATOKEN = 0
+OUTPUT_COST_PER_GIGATOKEN: int = 0
+INPUT_COST_PER_GIGATOKEN: int = 0
+# In USD
+# None = No limit
+MAX_CREDIT_USAGE_PER_MONTH: float | None = None
```

Comment:
Do you think USD would be a better unit here? You suggested setting `100M` credits here but to folks running the server, USD will be a simpler unit to use.

Reply:
Yeah, `MAX_PER_USER_MONTHLY_AI_COST: float = 0.50` is probably a better example value, yeah.

Reply:
Makes sense, changed!